pypeman/channels.py

```diff
@@ -935,9 +935,14 @@ def _handle_callback(self, future):
         except Dropped:
             pass
 
+    async def infinite_watcher(self):
+        while not self.is_stopped():
+            await self.watch_for_file()
+            await self.interruptable_sleeper.sleep(self.interval)
+        logger.warning("Won't watch anymore")
```

Comment:
"Won't watch anymore" is a completely true message. but for somebody less knowlegable about pypeman this might not be that helpful. 

And why is this a warning??

wouldn't logger.info("stopped watcher %s", repr(self)) be more helpful?

Reply:
changed